### ISSUE TITLE
Getting arround and arround the same issues and my tokens are reducing

### ISSUE THREAD

**Issue by federbalboa** [stackblitz/bolt.new#2018]
### Describe the bug

Im getting less tokens to fix issues that I have already fixed, and is consuming tokens to get the same work that has been already build

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-x4vjkr

### Steps to reproduce

im repeting the work I already build to fix and rework the same, 

### Expected behavior

keep the build already done, and do only the changes when is it needed

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by endocytosis** [comment]
Appreciate the feedback! If you have a file at a state you are satisfied with, you can right-click that file in the Bolt code editor and select "Lock file" to prevent the AI from making further changes to that file. This can be done for multiple files. This thread [here](https://github.com/stackblitz/bolt.new/issues/678) contains insights on using Bolt most effectively as well as ways the team is improving token efficiency. Hope this helps!

**Comment by federbalboa** [comment]
Thank you for the information, this will be helpful just before starting
because I drawn my token out. and this now gives me more control, thank you
very much for this, have a great day
